src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/html.groovy

```diff
@@ -24,59 +24,36 @@ def l = namespace(lib.LayoutTagLib)
 def st = namespace("jelly:stapler")
 
 l.layout(title:_("Pipeline Syntax: Reference"), norefresh: true) {
-    st.include(page: 'sidepanel')
```

Comment:
Not clear what you are doing to this page, which IIUC should not have been affected.